pkg/api/testing/fuzzer.go

```diff
@@ -117,8 +117,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			c.FuzzNoCustom(j) // fuzz self without calling this function again
 		},
 		func(j *api.DaemonControllerStatus, c fuzz.Continue) {
```

Comment:
It fuzzes everything by default-- you only need to add functions if the
default behavior is wrong for some reason.

On Thu, Jul 9, 2015 at 9:56 AM, Ananya Kumar <notifications@github.com>
wrote:

> In pkg/api/testing/fuzzer.go
> <https://github.com/vmarmol/kubernetes/pull/2#discussion_r34276838>:
>
> > @@ -117,8 +117,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
> >  			c.FuzzNoCustom(j) // fuzz self without calling this function again
> >  		},
> >  		func(j *api.DaemonControllerStatus, c fuzz.Continue) {
>
> Oh yeah, I didn't understand the details of fuzz testing. I'll talk to you
> about this later IRL. Do values get automatically fuzzed? Also, I'm not
> sure why we don't fuzz the Spec.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vmarmol/kubernetes/pull/2/files#r34276838>.
>